src/AutoPilotPlugins/PX4/PX4PIDTuningComponentCopter.qml

```diff
@@ -0,0 +1,177 @@
+/*=====================================================================
+
+ QGroundControl Open Source Ground Control Station
+
+ (c) 2009 - 2015 QGROUNDCONTROL PROJECT <http://www.qgroundcontrol.org>
+
+ This file is part of the QGROUNDCONTROL project
+
+ QGROUNDCONTROL is free software: you can redistribute it and/or modify
+ it under the terms of the GNU General Public License as published by
+ the Free Software Foundation, either version 3 of the License, or
+ (at your option) any later version.
+
+ QGROUNDCONTROL is distributed in the hope that it will be useful,
+ but WITHOUT ANY WARRANTY; without even the implied warranty of
+ MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ GNU General Public License for more details.
+
+ You should have received a copy of the GNU General Public License
+ along with QGROUNDCONTROL. If not, see <http://www.gnu.org/licenses/>.
+
+ ======================================================================*/
+
+import QtQuick              2.5
+import QtQuick.Controls     1.4
+
+import QGroundControl.Controls  1.0
+
+FactStepperGroup {
+    anchors.fill: parent
+    panelTitle:   "PID Tuning"
+    pageDescription: "PID Tuning is used to setup Proportional Integral Derivative controllers"
+
+    steppersModel: ListModel {
+
+///////////////////////////////////////////////////////////////
+        ListElement {
+            title:          qsTr("Roll P")
+            param:          "MC_ROLL_P"
+            group:          "Roll"
```

Comment:
Could you put these group names into a readonly string property so that they aren't repeated over and over again?